features/attached_enable.feature

```diff
@@ -266,3 +269,55 @@ Feature: Enable command behaviour when attached to an UA subscription
             FIPS enabled
             A reboot is required to complete install
             """
+
+    @series.bionic
+    @uses.config.machine_type.lxd.vm
+    Scenario: Attached enable fips on a machine with livepatch active
+        Given a `bionic` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        Then stdout matches regexp:
+            """
+            Updating package lists
+            ESM Infra enabled
+            Installing canonical-livepatch snap
+            Canonical livepatch enabled
+            """
+        When I run `ua disable livepatch` with sudo
+        Then I will see the following on stdout:
+            """
+            Removing canonical-livepatch snap
+            """
+        When I run `ua enable fips --assume-yes --beta` with sudo
```

Comment:
Can we also add a check that "which canonical-livepatch" returns empty string after the removal